binance/client.py

```diff
@@ -12,7 +12,7 @@
 class Client(object):
 
     API_URL = 'https://api.binance.{}/api'
-    WITHDRAW_API_URL = 'https://api.binance.{}/wapi'
+    WITHDRAW_API_URL = 'https://api.binance.{}/sapi'
```

Comment:
I think we should delete all `HTML` endpoints and functions because they are coping resgular SAPI endpoints...
This will be better but `breaking changes`.
Or we could deleta all `WAPI` infrastucture from code except `WAPI` functions but rewrite this function to call `SAPI` functions.
What do you think?

Reply:
would you kindly open up a new pr? i can review for you

Reply:
I'll try tomorrow.